documentation/modules/cruise-control/proc-generating-optimization-proposals.adoc

```diff
@@ -125,7 +125,23 @@ Status:
 ----
 +
 The properties in the `Optimization Result` section describe the pending cluster rebalance operation. 
-For descriptions of each property, see xref:contents-optimization-proposals[Contents of optimization proposals]. 
+For descriptions of each property, see xref:contents-optimization-proposals[Contents of optimization proposals].
+
+.Insufficient CPU capacity
+
+If the Kafka cluster is overloaded in terms of CPU utilization and needs some additional brokers, you may see an error like the following in the `KafkaRebalance` status. This may happen because, while proposals exclude the `excludedTopics`, the CPU utilization still considers replicas of the `excludedTopics`.
+
+[source,shell,subs="+quotes"]
+com.linkedin.kafka.cruisecontrol.exception.OptimizationFailureException:
+        [CpuCapacityGoal] Insufficient capacity for cpu (Utilization 615.21,
+        Allowed Capacity 420.00, Threshold: 0.70). Add at least 3 brokers with
+        the same cpu capacity (100.00) as broker-0. Add at least 3 brokers with
+        the same cpu capacity (100.00) as broker-0.
+
+[NOTE]
+====
+These numbers do not make sense if you look at `Kafka.spec.kafka.resources` because Strimzi generates a CPU percentage Cruise Control configuration, instead of a CPU core configuration. It is like having a single "virtual" CPU which has the cycles of 1 or more CPUs (the number configured in `Kafka.spec.kafka.resources.limits.cpu`). This is counterintuitive, but it has no effect on the rebalance behavior since the ratio between the utilization and capacity remains the same.
```

Comment:
```suggestion
The error suggests adding brokers with CPU capacity shown as a percentage rather than CPU cores. CPU `resources` are configured as integers representing cores in Strimzi. You can ignore the percentages. This has no effect on the rebalance behavior, since the ratio between CPU utilization and capacity remains the same.
```

Reply:
I wasn't sure what were trying to say in this note. I found it difficult to relate to the error message. I've made a suggestion, assuming the main point is that we can ignore the numbers for capacity shown in the error.

Reply:
It reads better, but it is not quite right.